docs/pages/guides/_meta.js

```diff
@@ -1,9 +1,13 @@
 export default {
-  "replicating-onchain-state": "Replicating onchain state",
   "hello-world": "Hello World",
   "extending-a-world": "Extending a World",
   "adding-delegation": "Adding Delegation",
   "modules": "Writing MUD Modules",
   emojimon: "Emojimon",
   testing: "Testing",
+  "---": {
+    title: "", // no title renders as a line
+    type: "separator",
+  },  
+  "replicating-onchain-state": "Replicating onchain state",
```

Comment:
Looking at this with fresh eyes, it's really unclear why there's a separation:

<img width="289" alt="image" src="https://github.com/user-attachments/assets/713d4ce3-a216-4ae6-986d-77aa6e818cbf">